src/extension/app/store/app.js

```diff
@@ -700,22 +703,34 @@ export class AppStore {
    * Displays a toast message
    * @param {string} message The message to display
    * @param {string} [variant] The variant of the toast (optional)
+   * @param {function} [closeCallback] The close callback function
+   * @param {function} [actionCallback] The action callback function
+   * @param {string} [actionLabel] The action label
    * @param {number} [timeout] The timeout in milliseconds (optional)
    */
-  showToast(message, variant = 'info', timeout = 6000) {
-    const existingToast = this.sidekick?.shadowRoot?.querySelector('theme-wrapper').querySelector('toast-container');
-    if (existingToast) {
-      existingToast.remove();
+  showToast(message, variant = 'info', closeCallback = undefined, actionCallback = undefined, actionLabel = 'Ok', timeout = 3000) {
```

Comment:
Is the `= undefined` really needed?

Reply:
ESLint wants it :)

![Screenshot 2024-04-25 at 1 02 28 PM](https://github.com/adobe/aem-sidekick/assets/3231084/5d6386eb-b3e3-47bf-9224-a4ecb3fa2f54)